src/compiler/linker.rs

```diff
@@ -0,0 +1,9 @@
+use crate::ir::{sc::SuperCircuit, Circuit};
+
+pub struct Linker {}
+
+impl Linker {
+    pub fn link<F>(units: Vec<Circuit<F>>) -> SuperCircuit<F> {
+        let sc = SuperCircuit::default();
+    }
+}
```

Comment:
This file isn't linked to the module tree. Need to `pub mod linker` in compiler.rs.

Reply:
I see that it's probably not completed yet, so that's why we aren't adding it to the module tree.

Reply:
I will remove this file.

Reply:
Done